detection_rules/rule.py

```diff
@@ -361,6 +361,47 @@ def process_note_plugins():
 
         return obj
 
+    @validates_schema
+    def validate_bbr(self, value: dict, **kwargs):
+        def convert_time_span(span: str) -> int:
+            """Convert time span in datemath to value in milliseconds."""
```

Comment:
since this is repeated, I would just make it a function in `utils` and then call that from this method as well as the `EQLRuleData` method

Reply:
moved to `utils.py`